src/Service.Host/client/src/containers/parts/Parts.js

```diff
@@ -6,13 +6,16 @@ import partsSelectors from '../../selectors/partsSelectors';
 import { getPrivileges } from '../../selectors/userSelectors';
 import partTemplatesActions from '../../actions/partTemplatesActions';
 import partTemplatesSelectors from '../../selectors/partTemplatesSelectors';
+import partTemplateActions from '../../actions/partTemplatesActions';
+import partTemplateSelectors from '../../selectors/partTemplatesSelectors';
 
```

Comment:
do we need these extra action creators and selectors in the parts container?

Reply:
I also just realised I never got you set up witht the linter which would have complained about this most likely. Give me a shout today and I can try and remember what we need to do to get it working

Reply:
Can talk about it after stand up if you're about